### DISCUSSION TITLE
unable to create ticket and to create service

### DISCUSSION THREAD

**Discussion by JOduMonT** [ovh/manager#6542] [Q&A]
I have an account with OVH since 2015; but recently, I'll say for 5 months, I'm unable to create any services and since ticket have to be created link to a service I'm unable to create a ticket to declare that issue.

My credit card is valid, my email for service and bill are valid, my phone number is valid.
here is a concrete example when I want to create a new public cloud project

```
Oops …!
Une erreur est survenue lors du chargement de la page.
Internal server error
Code d'erreur : CA.manager-2.6215c29e.32684.5f12a89b-d510-462c-831e-165a94817d28

```
![image](https://user-images.githubusercontent.com/5204724/155265225-5ce7e9b3-7f01-4656-abc2-20fb21573908.png)

**Comment by bitsn1per** [comment]
Yes, there is no attention for the customer's problems. I went here just because I was searching additional way to contact support. But now I can see, that OVH is don't bothers about their client's needs. I'm trying to pay for my VPS - no one can help me, in chat or via tickets, and even when I decided to pay for new VPS - they just took my money and I still have no access to it. What the heck is going on?

**Reply by JOduMonT** [reply]
It is unfortunate, I found a way to interact with someone; as I create a new account with the same email and password :P
- _which is weird and should not being authorized_ - and now they “support me”, as they answer my request but still, during the meanwhile of them “trying” to resolve my issue (this time, I believe it is because I'm using a credit card with OTP); I had time to create account at ScaleWay and Hetzner and build my VMs, even I used the same credit card. ;)

**Comment by JeremyDec** [comment]
Hello @JOduMonT,

I have tried to access to ticket creation through the link in the user panel inside the Control Panel, with your nic account.

I was able to select option to indicate that no service exists yet, see the following screen:

![image](https://user-images.githubusercontent.com/34273689/156721200-d8ef43ae-94f1-4b27-82c0-4d755b52237c.png)

Is that what you're looking for?

Regards,
Jeremy
